packages/queries/src/types/QueryProviderInput.ts

```diff
@@ -0,0 +1,19 @@
+import { ColumnValue } from './ColumnValue';
```

Comment:
I'm also thinking that this type shouldn't be in /queries, it should be in /backend as you would only ever use this type in a backend context.